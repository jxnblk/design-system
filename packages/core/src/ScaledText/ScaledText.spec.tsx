import React from 'react'
import { ScaledText } from '..'

describe('ScaledText', () => {
  test('renders svg text nodes in parent container', () => {
    const json = rendererCreateWithTheme(
      <div>
        <ScaledText
          values={[{ value: 'Hello', props: { fontWeight: 'bold' } }, { value: 'World' }]}
          color='red'
        />
      </div>
    ).toJSON()
    expect(json).toMatchSnapshot()
  })
})
