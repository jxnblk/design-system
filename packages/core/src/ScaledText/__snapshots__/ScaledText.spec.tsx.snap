// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScaledText renders svg text nodes in parent container 1`] = `
<div>
  <svg
    className="svg-text-8f386643-cc56-4ab8-8d92-0d599f7fd548"
  >
    <text
      fill="red"
      fontWeight="bold"
      title="Hello"
    >
      Hello
    </text>
  </svg>
  <svg
    className="svg-text-8f386643-cc56-4ab8-8d92-0d599f7fd548"
  >
    <text
      fill="red"
      title="World"
    >
      World
    </text>
  </svg>
</div>
`;
